homeassistant/components/samsungtv/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
```

Comment:
Why do we check for `self.hass.is_stopping` in the coordinator?

In the code of the coordinator we already check that
```py
        if self._shutdown_requested or (scheduled and self.hass.is_stopping):
            return

        if log_timing := self.logger.isEnabledFor(logging.DEBUG):
            start = monotonic()

        auth_failed = False
        previous_update_success = self.last_update_success
        previous_data = self.data

        try:
            self.data = await self._async_update_data()
```

Reply:
Resolved in #143949